src/driver/test/Driver_test.cpp

```diff
@@ -155,6 +156,7 @@ std::pair<DefaultDriver::OutcomeCode, DefaultDriver::ProgramOptions> outcome(
 
 
 TEST_CASE("Various Actor Behaviors") {
+    boost::filesystem::current_path(genny::findRepoRoot());
```

Comment:
do you need this?

Reply:
ooh..I missed that this sets cwd....hm....that seems like it could make tests order-dependent. Fine with this for now but I'd prefer tests to never assume cwd and call `findRepoRoot` on their own if they need to access a file at a known location.

Reply:
Yeah I had intended to set the CWD for this `TEST_CASE` only. Other test cases will have to set their own to avoid order dependencies.